app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,62 @@
  *= require_tree .
  *= require_self
  */
+@import url('https://fonts.googleapis.com/css2?family=Lobster&display=swap');
+
+body {
+  font-family: proxima-nova, sans-serif !important;
```

Comment:
- [ ] The `!important` CSS  attribute is **overly used** in your project. Kindly ensure that the attribute is used sparingly.